omicsclaw.py

```
@@ -312,25 +309,28 @@ def _handle_auth_command(args) -> None:
 
 
 def list_skills(domain_filter: str | None = None) -> dict:
-    """按 Domain 分组打印所有可用技能，并返回 SKILLS 字典。"""
+    """按 Domain 分组打印所有可用技能，并返回 skills 字典。"""
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)